deploy-service/common/src/main/java/com/pinterest/deployservice/rodimus/RodimusManagerImpl.java

```diff
@@ -68,31 +73,97 @@ public boolean shouldSkipClass(Class<?> c) {
         }
     }
 
+    private boolean refreshCachedKey() throws Exception {
+        String prevKnoxKey = this.catchedKey;
```

Comment:
typo in catchedKey?

Reply:
it is a typo... replaced with **cached**... also _catched_ as a word does not exist